link/kprobe.go

```diff
@@ -49,6 +49,7 @@ type KprobeOptions struct {
 	//
 	// Needs kernel 5.15+.
 	Cookie uint64
+	Offset uint64
```

Comment:
Could you add a comment which describes what this is?

Reply:
ok